### PR TITLE
all ACM edits for Background, with some change questions

### PR DESCRIPTION
**Pull request by acmacm**
See my marked-up text file for additional info on the changes (comments/questions).

### CODE REVIEW

File: src/background.xml [hunk 1]
```diff
@@ -5,66 +5,76 @@
       Transport Capacity measurement was known to be well beyond
       our capabilities. Even at the time that Framework for IP
       Performance Metrics 
-      <xref target="RFC3148" /> was written we knew that we didn't
```
Comment (mattmathis) [inline src/background.xml:8]: No I meant 3148, although we knew even less when 2330 was written.

File: src/background.xml [hunk 2]
```diff
@@ -5,66 +5,76 @@
...
       <list style="symbols">
 
         <t>TCP is a control system with circular dependencies -
         everything affects performance, including components that
-        are explicitly not part of the test.</t>
+        are explicitly not part of the test (for example, the host 
+        processing power is not in-scope of path performance tests).</t>
 
-        <t>Congestion control is an equilibrium process, such that
-        transport protocols change the packet transfer statistics
```
Comment (mattmathis) [inline src/background.xml:18]: I meant "equilibrium process" as in https://en.wikipedia.org/wiki/Dynamic_equilibrium.  I quote "Reactions do in fact occur, sometimes vigorously, but to such an extent that changes in composition cannot be observed." (talking chemistry, where the phenomena was first studied) 

"seeking equilibrium" to most people means asymptotically converging to zero slope or a fixed value, which is true but not the point I was trying to make.

File: src/background.xml [hunk 3]
```diff
@@ -5,66 +5,76 @@
...
+        capacity of a connection, (e.g. the bottleneck is 100% utilized),
+        then packet losses and ECN CE marks are mostly determined by 
+        the TCP sender and how aggressively it seeks additional capacity 
+        from the network, not the network itself.</t>
+
+        <t>TCP's ability to compensate for network impairments is
+        directly proportional to the number of send-ACK roundtrip exchanges per
         second (i.e. inversely proportional to the RTT). As a
-        consequence a flawed subpath may pass a short RTT local
+        consequence a subpath with limiting bottleneck may pass a short RTT local
```
Comment (mattmathis) [inline src/background.xml]: No.  

How about s/flawed/high packet loss rate/  The point is the network is NOT full but has spurious losses that look like congestion.   Such a link is a show stopper at long RTTs and inconsequential a short RTT's.

**Comment by mattmathis** [comment]
I don't understand commenting.  Can you see my inline comments?

Two general comments:  For inband comments to authors please tag with @@@@ so they are searchable and stand out in printed text.   Initials might be nice in some cases.  e.g.
(@@@@ I have a problem with this passage - MM)

File: src/background.xml (hunk 1 above)
Comment (acmacm) [inline src/background.xml:8]: Then the title of 2330, "Framework for IP Performance Metrics", shouldn't appear in the text. Maybe "Empirical BTC Metrics" instead.

File: src/background.xml (hunk 2 above)
Comment (acmacm) [inline src/background.xml:18]: Maybe add more explanation at the end of the original sentence:
...protocols change the packet transfer statistics (raise the packet
   loss ratio and/or RTT) to conform to their behavior (e.g. Dynamic Equilibrium [ref to wikipedia of other source] ).

File: src/background.xml (hunk 3 above)
Comment (acmacm) [inline src/background.xml]: I didn't get your point of emphasis - these are network impairments outside of those caused by TCP itself.
It's not just loss, network impairments also include delay and delay variation experienced on the connection which reduce TCP's ability to compensate.. 
If I can return to the right branch, I'll suggest:

   TCP's ability to compensate for network impairments  (such as loss, delay and delay variation,  outside of those caused by TCP itself)  is directly
      proportional to the number of send-ACK roundtrip exchanges per second (i.e.
      inversely proportional to the RTT).  As a consequence an impaired
      subpath may pass a short RTT local test even though it fails when
      the subpath is extended by an effectively perfect network to some
      larger RTT.

**Comment by acmacm** [comment]
I’m working from the attached file, essentially
implementing edits suggested there. I have tried to
transfer the comments when some background might help,
but mostly in the pull-request comment field.

In the file, all edits and comments can be found at “[ACM]”.

Question:
Is there some easy way to map the .xml filenames to sections?
When I got to section 4 and 5, finding the right file became
difficult – had to perform text searches on the src directory.

Al

**Pull request merged**